src/app.md

```diff
@@ -632,6 +633,53 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-semver-complex]: #ex-semver-complex
+<a name="ex-semver-complex"></a>
+## Parse a complex version string.
+
+[![semver-badge]][semver] [![cat-config-badge]][cat-config]
+
+Constructs a [`semver::Version`] from a complex version string using [`Version::parse`]. The string
+contains pre-release and build metadata as defined in the [Semantic Versioning Specification].
+
+Note that, in accordance with the Specification, build metadata is not considered when comparing
+versions. In other words, two versions may be equal even if their build strings differ.
+
+```rust
+# #[macro_use]
+# extern crate error_chain;
+extern crate semver;
+
+use semver::{Identifier, Version};
+#
+# error_chain! {
+#     foreign_links {
+#         SemVer(semver::SemVerError);
+#     }
+# }
+
+fn run() -> Result<()> {
+    let version_str = "1.0.49-125+g72ee7853";
+    let parsed_version = Version::parse(version_str)?;
+
+    let constructed_version = Version {
+        major: 1,
+        minor: 0,
+        patch: 49,
+        pre: vec![Identifier::Numeric(125)],
+        build: vec![],
+    };
+    assert_eq!(parsed_version, constructed_version);
+
```

Comment:
lets reintroduce the separate assert_eq with just the build metadata to show that it is in fact parsed.